submissions/onyemeMichael/php-db/cookies.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+require_once("connection.php");
+
+echo "<h3> Customer Details </h3>";
+
+//Querying the database to select all information available in the table
+class QueryDb extends Connect {
+    public function customer() {
+        try {
+            $sql= "SELECT * FROM customer_details";
+            $stmt = $this->conn()->query($sql);
+            $result = $stmt->fetchAll();
+            
+        // setting up the cookie to save the result in the databsae
+        setcookie("customer", serialize($result), time()+7200, "/");
+        return $result;
+        } catch (PDOException $e) {
+            die ('could not execute ' . $sql . $e->getMessage());
+        }
+    }
+}
+//To call the customer function
+$user = new QueryDb();
+$result = isset($_COOKIE['customer']) ? unserialize($_COOKIE['customer']) : $user->customer();
+
+    if(count($result) > 0)  {
+        echo "<table>";
+            echo "<tr>";
+                echo "<th> S/N </th>";
+                echo "<th> Full Name </th>";
+                echo "<th> Email Address </th>";
+                echo "<th> Created At</th>";
+                echo "<th> Actions </th>";
+            echo "</tr>";
+            foreach($result as $row) {
+                echo "<tr>";
+                    echo "<td>" . $row['id'] . "</td>";
+                    echo "<td>" . $row['full_name'] . "</td>";
```

Comment:
There is no column called `full_name` on the customers table
![image](https://user-images.githubusercontent.com/43061141/106275573-59d7c100-6236-11eb-8905-0b3afd3207b2.png)